src/client/gui/lib/platform/windows.dart

```diff
@@ -33,8 +33,12 @@ class WindowsPlatform extends MpPlatform {
   Map<SingleActivator, Intent> get terminalShortcuts => const {
         SingleActivator(LogicalKeyboardKey.keyC, control: true, shift: true):
             CopySelectionTextIntent.copy,
+        SingleActivator(LogicalKeyboardKey.insert, control: true):
+            CopySelectionTextIntent.copy,
         SingleActivator(LogicalKeyboardKey.keyV, control: true, shift: true):
             PasteTextIntent(SelectionChangedCause.keyboard),
+        SingleActivator(LogicalKeyboardKey.insert, shift: true):
+            PasteTextIntent(SelectionChangedCause.keyboard),
```

Comment:
I suppose they can be added to Linux as well. My impression was that they were more common on Windows. But on macOs I don't think it makes much sense, since `Insert` is not a standard key.

Reply:
I see, I didn't know macOS doesn't have `Insert`. That makes more sense, as for Linux I'll leave that up to you.

Reply:
alright, i added them to linux as well. they seem to be common enough.

Reply:
FWIW, I use shift+insert on Linux all the time.